x-pack/plugins/integrations_manager/server/requests.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
I think it probably makes sense to standardize on the await syntax except for scenarios where promises and then are necessary?

Reply:
Updated in e221798 since we're not wrapping a callback-based lib any longer. Thanks.